src/control/lib/daos/system_prop_test.go

```diff
@@ -180,22 +180,19 @@ func TestDaos_SystemPropertyKeys(t *testing.T) {
 }
 
 func TestDaos_SystemPropertySelfHealHasFlag(t *testing.T) {
-	for name, tc := range map[string]struct {
-		value string
-		flag  SelfHealFlag
```

Comment:
why the revert?

Reply:
sorry that's annoying. I'm not suggesting a change just wondering why you are switching back from table -based tests?